BUILD.gn

```diff
@@ -677,8 +722,10 @@ if (is_mac) {
       ":electron_lib",
     ]
     deps = [
+      ":electron_angle_library",
```

Comment:
Any reason to isolate these in targets like `electron_angle_library`, `electron_swiftshader_library` ? Can't they be combined into `electron_framework_libraries` ?

Reply:
I just copy/pasted this from content_shell. Left it the same because that was easiest 🤷‍♂